h2o-algos/src/main/java/hex/gam/GAMModel.java

```diff
@@ -358,6 +364,7 @@ public double linkInv(double x) {
     public double[][] _oneOGamColStd;
     public double[] _penaltyScale;
     public Key<Frame> _gamTransformedTrainCenter;  // contain key of predictors, all gamified columns centered
+    public Key<Frame>[] _gamTrainsformedTrain;  // key of predictors, not centered, for testing only
```

Comment:
nit: typo

Reply:
I cannot find the typo.  Please help me.

Reply:
I no longer see this diff, so no issue